### ISSUE TITLE
Get rid of the textfield's workaround 

### ISSUE THREAD

**Issue by yuriy-fix** [vaadin/vaadin-select#194]
Get rid of the textfield's template workaround: https://github.com/vaadin/vaadin-select/commit/e7da830e7808342836018453ef153ef07b787186. 
Requires the bump of the text-field dependency.

**Comment by TatuLund** [comment]
Note, due to thing mentioned, you need to use different selector for styling value than with normal vaadin-text-field

```
:host([theme~="custom-select"]) [part="input-field"] ::slotted([part="value"]) {
   background: blue;
}
```

**Comment by web-padawan** [comment]
This was fixed in #234. We always use `input` slot now.
